src/DataInterpolations.jl

```diff
@@ -31,6 +31,13 @@ ChainRulesCore.frule((_, _, Δt), ::typeof(_interpolate), A::AbstractInterpolati
 
 (interp::AbstractInterpolation)(t::Number) = _interpolate(interp, t)
 
+import Symbolics
+Base.@pure __parameterless_type(T) = Base.typename(T).wrapper
```

Comment:
What is this function used for?

Reply:
`Array{Float64} -> Array` for the op name.

Reply:
But where is it used?

Reply:
good point.